docs/aerodromes/classd/Archerfield.md

```diff
@@ -69,6 +69,64 @@ The circuit direction changes depending on time of day and runway being used.
+    Both the **Northern** and **Southern** helipads are inside the maneuvering area and treated like a runway, requiring a takeoff and landing clearance from AF ADC. Helicopters operating on other surfaces outside the maneuvering areas should be instructed to *"report airborne"*  or *"report on the ground"* so the controllers knows when they are active / no longer active in their airspace. 
+
+### Training Areas
+There are 3 designated helicopter training areas at Archerfield:
+
+**Area A**: North of Taxiway B within the confines of the aerodrome boundary, clear of all buildings, not above `A005`. Circuit direction and frequency as per ATIS. A maximum of 3 helicopters are permitted to operate circuits concurrently. Additionally, a maximum of 2 helicopters are permitted to operate in ground effect only.
+
+**Area B**: North and west of Runway 04L within the confines of the aerodrome boundary, clear of all buildings, not above `A005`. Circuit direction and frequency as per ATIS. A maximum of 3 helicopters are permitted to operate circuits concurrently. Additionally, a maximum of 2 helicopters are permitted to operate in ground effect only.
+
+**Area C**: South and east of Taxiway D, inside the fixed wing circuit for runway 04R/22L, east of the control tower, not above `A008`. Landings must be conducted north of Taxiway B. Once established, the only radio communications are an airborne call before each circuit and a downwind call by pilots. A maximum of 2 helicoptesr are permitted to operate in this area concurrently.
+
+<figure markdown> 
+  ![Archerfield Helicopter Training Areas](img/ybaf_helicopter_area.png){ width="450" }
+  <figcaption>Archerfield Helicopter Training Areas</figcaption>
+</figure>
+
+
+### Departures
+Helicopters comply with fixed wing outbound procedures. Departures shall be parallel to and clear of the duty runways. Departures with intended entry into BN CTR shall advise AF SMC on taxi with "Uni Departure". No heads-up coordination with BN TCU is required.
+
+!!! phraseology 
+    **TUY:** "Archer Tower, helicopter TUY, northen pad, for eastern departure, ready"  
+    **AF ADC:** "TUY, Archer Tower, northen pad, cleared for takeoff"  
+    **TUY:** "Cleared for takeoff, northen pad, TUY"  
+
+### Arrivals
+Helicopters comply with fixed wing inbound procedures. Pilots shall nominate the required HLS on first contact with AF ADC. Helicopters may be instructed to overfly the runway complex not below `A005` for a short circuit to the designated HLS.
+
+!!! phraseology 
+    **82Y:** "Archer Tower, helicopter 82Y, UQLD A010, recieved Foxtrot, inbound northern pad"  
+    **AF ADC:** "82Y, Archer Tower, track direct northen pad, report 1 mile"  
+    **82Y:** "Track direct northern pad, 82Y"  
+
+    **82Y:** "82Y, 1 mile, northern pad"  
+    **AF ADC:** "82Y, cleared to land northern pad, no requirements with ground, taxi to parking"  
+    **82Y:** "Cleared to land northen pad, remaining your frequency, taxi to parking, 82Y"
```

Comment:
Is the 'no requirements with ground' procedure used every time, or just when traffic is light?

Reply:
Seems to be a procedure used every time after listening to ADC a many times. The air taxi from the northern pad to the various helicopter company hangars is so short  that ADC will say "no requirements with ground". Additionally, any helicopters which have nominated their HLS as the northern pad can be assumed to be parking at the hangars next to it, otherwise they would have nominated the central pad instead.

Reply:
Fair enough, would make more sense for a VATSIM environment.

Reply:
Fixed

Reply:
Fixed